python/ray/tests/test_multi_node.py

```diff
@@ -317,7 +317,7 @@ def test_calling_start_ray_head():
             "ray", "start", "--head", "--redis-port", "6379",
             "--redis-shard-ports", "6380,6381,6382", "--object-manager-port",
             "12345", "--num-cpus", "2", "--num-gpus", "0",
-            "--redis-max-clients", "100", "--resources", "{\"Custom\": 1}"
+            "--redis-max-clients", "100", "--resources", "{\'Custom\': 1}"
```

Comment:
I don't think this change is correct. We shouldn't make sure the json parser treats `\'`  as  ` \" `.

I think the best approach to fix the CI failure is disable the check right now, then enable the quote check which can allow this statement `"\"abc\""` and reject this kind statement `'abc'`.